server/src/constants/index.ts

```diff
@@ -0,0 +1,34 @@
+export const AVAILABLE_SUBSCRIPTION_PLANS = [
+  {
+    type: 'superPlan',
+    title: 'Super Plan',
+    priceWithSale: 1100,
+    priceWithoutSale: 1500,
+    duration: '1 month',
+    currency: 'USD',
+  },
+  {
+    type: 'megaPlan',
+    title: 'Mega Plan',
+    priceWithSale: 2200,
+    priceWithoutSale: 3500,
+    duration: '1 month',
+    currency: 'USD',
+  },
+  {
+    type: 'superMegaPlan',
+    title: 'Super Mega Plan',
+    priceWithSale: 3300,
+    priceWithoutSale: 4700,
+    duration: '1 month',
```

Comment:
using strings to represent numbers is almost never a good idea, types that represent time should be time or number types. in this case, i'd recommend duration to be either a `moment` type or durationInDays. and if you want to represent eternity, you can also just use inf OR you can just use an absurdly high integer, which is slightly hacky but fine.